docs/elements.md

```diff
@@ -275,6 +277,52 @@ Attribute | Type | Default | Description
 
 -----
 
+## `pl-dropdown` element
+
+Select the correct answer from a drop-down **select** menu list of potential answers. The potential options are listed in the inner HTML of a <pl-answer></pl-answer> element (ie. <pl-answer>Possible Answer 1</pl-answer>).
+
+#### Sample Element
+
+![](elements/pl-dropdown.png)
+
+**question.html**
+
+```html
+An <pl-dropdown answers-name="socrates">
+  <pl-answer correct="true">unexamined</pl-answer>
+  <pl-answer>examined</pl-answer>
+  <pl-answer>insatiable</pl-answer>
+</pl-dropdown> life is not worth living.
+```
+
+**server.py**
+```python
+def generate(data):
+
+    # Correct Answers
+    data['params']['socrates'] = {
+      'tag1': 'true', 'ans1': 'whole',
+      'tag2': 'false', 'ans2': 'part',
+      'tag3': 'false', 'ans3': 'inverse'
+    }
+```
+
```

Comment:
I think we can remove this server code, since we're putting the answer choices straight in the example html?

Reply:
I happened to just be cleaning that part up. Can you take a look now and see what you think? It should be consistent.

Reply:
Can we have at least one section of the `elements/dropdown` question totally defined in the `question.html` file?  As in taken out of the `server.py` and just static in the html.  I am sure people will be copy pasting from this file to use in their own questions and would like them to be aware that they can create dropdowns without the server file.

Reply:
Most definitely. That sounds good!

Reply:
I took out the second example to simplify this example. We have one example entered through the HTML and one example entered through the server.py file.

![image](https://user-images.githubusercontent.com/7872295/85878479-9cf82000-b78d-11ea-9185-9854e7817c72.png)